### ISSUE TITLE
menu: showVisitedLinks not working for some theme variants

### ISSUE THREAD

**Issue by McShelby** [McShelby/hugo-theme-relearn#518]
### Discussed in https://github.com/McShelby/hugo-theme-relearn/discussions/517

<div type='discussions-op-text'>

<sup>Originally posted by **skiboyjcmu** May  3, 2023</sup>
my team really likes the showVisitedLinks feature in the menu to help indicate to participants that they've completed (or at least visited) a task/chapter.  Having recently migrated from Learn to ReLearn theme, I'm finding the check marks aren't showing up in certain Themes.  You can replicate this with the [reLearn theme page](https://mcshelby.github.io/hugo-theme-relearn/index.html) and the variant switcher at the bottom of nav menu

Checkmarks show up on the Relearn Light | Brighti | Neon, but not on any of the other variants.  I haven't been able to identify why it's behaving this way.  Any ideas?

![image](https://user-images.githubusercontent.com/20427852/236003680-b51a881f-5dd2-49a6-a628-6bc3f3e93d80.png)
</div>

**Comment by McShelby** [comment]
Currently only light, bright and neon seem to work. Really puzzling

**Comment by skiboyjcmu** [comment]
> Currently only light, bright and neon seem to work. Really puzzling

Actually, I just realized that the checkmarks DO appear for past VisitedLinks which are not the currently selected chapter.  Also I've seen them show up briefly when switching themes, only to be immediately painted over (not sure what the technical term is, but it's as if they are there in a color, and then just being overwritten by the black background).

So I've visited all of chapter 1 & 2, and some of 3.

When I'm in a chapter 2 page, the chapter 1&3 visited checks show up: 
![image](https://user-images.githubusercontent.com/20427852/236296281-b7f997f3-5bdf-4b67-9f04-d8c040afa1be.png)

When I'm in a Chapter 1 page, the chapter 1&3 visited checks show up: 

![image](https://user-images.githubusercontent.com/20427852/236296380-06f69f8a-039d-4b7c-9610-06108fbb37d5.png)
